{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting tensorflowjs\n",
      "  Downloading tensorflowjs-3.12.0-py3-none-any.whl (77 kB)\n",
      "Collecting tensorflow-hub<0.13,>=0.7.0\n",
      "  Downloading tensorflow_hub-0.12.0-py2.py3-none-any.whl (108 kB)\n",
      "Collecting tensorflowjs\n",
      "  Downloading tensorflowjs-3.11.0-py3-none-any.whl (64 kB)\n",
      "  Downloading tensorflowjs-3.9.0-py3-none-any.whl (64 kB)\n",
      "  Downloading tensorflowjs-3.8.0-py3-none-any.whl (64 kB)\n",
      "  Downloading tensorflowjs-3.7.0-py3-none-any.whl (64 kB)\n",
      "  Downloading tensorflowjs-3.6.0-py3-none-any.whl (63 kB)\n",
      "Collecting tensorflow-hub<0.10,>=0.7.0\n",
      "  Downloading tensorflow_hub-0.9.0-py2.py3-none-any.whl (103 kB)\n",
      "Collecting tensorflowjs\n",
      "  Downloading tensorflowjs-3.5.0-py3-none-any.whl (63 kB)\n",
      "Requirement already satisfied: h5py<3,>=2.8.0 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from tensorflowjs) (2.9.0)\n",
      "  Downloading tensorflowjs-3.4.0-py3-none-any.whl (63 kB)\n",
      "  Downloading tensorflowjs-3.3.0-py3-none-any.whl (63 kB)\n",
      "  Downloading tensorflowjs-3.2.0-py3-none-any.whl (63 kB)\n",
      "  Downloading tensorflowjs-3.1.0-py3-none-any.whl (63 kB)\n",
      "  Downloading tensorflowjs-3.0.0-py3-none-any.whl (63 kB)\n",
      "  Downloading tensorflowjs-2.8.5-py3-none-any.whl (63 kB)\n",
      "  Downloading tensorflowjs-2.8.4-py3-none-any.whl (63 kB)\n",
      "  Downloading tensorflowjs-2.8.3-py3-none-any.whl (63 kB)\n",
      "  Downloading tensorflowjs-2.8.2-py3-none-any.whl (63 kB)\n",
      "  Downloading tensorflowjs-2.8.1-py3-none-any.whl (63 kB)\n",
      "  Downloading tensorflowjs-2.8.0-py3-none-any.whl (63 kB)\n",
      "  Downloading tensorflowjs-2.7.0-py3-none-any.whl (62 kB)\n",
      "  Downloading tensorflowjs-2.6.0-py3-none-any.whl (61 kB)\n",
      "  Downloading tensorflowjs-2.5.0-py3-none-any.whl (61 kB)\n",
      "  Downloading tensorflowjs-2.4.0-py3-none-any.whl (61 kB)\n",
      "Collecting numpy<1.19.0,>=1.16.4\n",
      "  Downloading numpy-1.18.5-cp37-cp37m-win32.whl (10.8 MB)\n",
      "Requirement already satisfied: six>=1.12.0 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from tensorflowjs) (1.12.0)\n",
      "Collecting PyInquirer==1.0.3\n",
      "  Downloading PyInquirer-1.0.3.tar.gz (27 kB)\n",
      "  Preparing metadata (setup.py): started\n",
      "  Preparing metadata (setup.py): finished with status 'done'\n",
      "Collecting tensorflowjs\n",
      "  Downloading tensorflowjs-2.3.0-py3-none-any.whl (61 kB)\n",
      "  Downloading tensorflowjs-2.1.0-py3-none-any.whl (60 kB)\n",
      "  Downloading tensorflowjs-2.0.1.post1-py3-none-any.whl (60 kB)\n",
      "  Downloading tensorflowjs-2.0.1-py3-none-any.whl (60 kB)\n",
      "Requirement already satisfied: numpy>=1.16.4 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from tensorflowjs) (1.21.4)\n",
      "  Downloading tensorflowjs-2.0.0-py3-none-any.whl (57 kB)\n",
      "  Downloading tensorflowjs-1.7.4.post1-py3-none-any.whl (57 kB)\n",
      "  Downloading tensorflowjs-1.7.4-py3-none-any.whl (57 kB)\n",
      "  Downloading tensorflowjs-1.7.3-py3-none-any.whl (57 kB)\n",
      "  Downloading tensorflowjs-1.7.2-py3-none-any.whl (57 kB)\n",
      "  Downloading tensorflowjs-1.6.0-py3-none-any.whl (57 kB)\n",
      "  Downloading tensorflowjs-1.5.2-py3-none-any.whl (56 kB)\n",
      "  Downloading tensorflowjs-1.4.0-py3-none-any.whl (56 kB)\n",
      "  Downloading tensorflowjs-1.3.2-py3-none-any.whl (53 kB)\n",
      "  Downloading tensorflowjs-1.3.1.1-py3-none-any.whl (53 kB)\n",
      "  Downloading tensorflowjs-1.3.1-py3-none-any.whl (53 kB)\n",
      "  Downloading tensorflowjs-1.2.10.1-py3-none-any.whl (47 kB)\n",
      "Collecting numpy==1.16.4\n",
      "  Downloading numpy-1.16.4-cp37-cp37m-win32.whl (10.0 MB)\n",
      "Collecting tensorflow-hub==0.5.0\n",
      "  Downloading tensorflow_hub-0.5.0-py2.py3-none-any.whl (78 kB)\n",
      "Collecting six==1.11.0\n",
      "  Downloading six-1.11.0-py2.py3-none-any.whl (10 kB)\n",
      "Collecting h5py==2.8.0\n",
      "  Downloading h5py-2.8.0-cp37-cp37m-win32.whl (2.0 MB)\n",
      "Collecting tensorflowjs\n",
      "  Downloading tensorflowjs-1.2.10-py3-none-any.whl (47 kB)\n",
      "  Downloading tensorflowjs-1.2.9-py3-none-any.whl (46 kB)\n",
      "Collecting keras==2.2.4\n",
      "  Downloading Keras-2.2.4-py2.py3-none-any.whl (312 kB)\n",
      "Collecting tensorflowjs\n",
      "  Downloading tensorflowjs-1.2.6-py3-none-any.whl (39 kB)\n",
      "  Downloading tensorflowjs-1.2.3-py3-none-any.whl (39 kB)\n",
      "  Downloading tensorflowjs-1.2.2.1-py3-none-any.whl (37 kB)\n",
      "Collecting tensorflow-hub==0.3.0\n",
      "  Downloading tensorflow_hub-0.3.0-py2.py3-none-any.whl (73 kB)\n",
      "Collecting tensorflowjs\n",
      "  Downloading tensorflowjs-1.2.2-py3-none-any.whl (38 kB)\n",
      "  Downloading tensorflowjs-1.2.1-py3-none-any.whl (37 kB)\n",
      "Collecting numpy==1.15.1\n",
      "  Downloading numpy-1.15.1-cp37-none-win32.whl (9.9 MB)\n",
      "Collecting tensorflowjs\n",
      "  Downloading tensorflowjs-1.1.2-py3-none-any.whl (36 kB)\n",
      "  Downloading tensorflowjs-1.0.1-py3-none-any.whl (35 kB)\n",
      "  Downloading tensorflowjs-0.8.6-py3-none-any.whl (39 kB)\n",
      "Collecting keras==2.2.2\n",
      "  Downloading Keras-2.2.2-py2.py3-none-any.whl (299 kB)\n",
      "Collecting tensorflowjs\n",
      "  Downloading tensorflowjs-0.8.5-py3-none-any.whl (39 kB)\n",
      "  Downloading tensorflowjs-0.8.0-py3-none-any.whl (39 kB)\n",
      "Requirement already satisfied: tensorflow==1.12.0 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from tensorflowjs) (1.12.0)\n",
      "Collecting tensorflow-hub==0.1.1\n",
      "  Downloading tensorflow_hub-0.1.1-py2.py3-none-any.whl (52 kB)\n",
      "Requirement already satisfied: scipy>=0.14 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from keras==2.2.2->tensorflowjs) (1.3.1)\n",
      "Collecting keras-preprocessing==1.0.2\n",
      "  Downloading Keras_Preprocessing-1.0.2-py2.py3-none-any.whl (26 kB)\n",
      "Requirement already satisfied: pyyaml in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from keras==2.2.2->tensorflowjs) (5.1.2)\n",
      "Collecting keras-applications==1.0.4\n",
      "  Downloading Keras_Applications-1.0.4-py2.py3-none-any.whl (43 kB)\n",
      "Requirement already satisfied: protobuf>=3.6.1 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from tensorflow==1.12.0->tensorflowjs) (3.19.1)\n",
      "Requirement already satisfied: wheel>=0.26 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from tensorflow==1.12.0->tensorflowjs) (0.33.6)\n",
      "Requirement already satisfied: grpcio>=1.8.6 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from tensorflow==1.12.0->tensorflowjs) (1.42.0)\n",
      "Requirement already satisfied: tensorboard<1.13.0,>=1.12.0 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from tensorflow==1.12.0->tensorflowjs) (1.12.2)\n",
      "Requirement already satisfied: gast>=0.2.0 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from tensorflow==1.12.0->tensorflowjs) (0.5.3)\n",
      "Requirement already satisfied: astor>=0.6.0 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from tensorflow==1.12.0->tensorflowjs) (0.8.1)\n",
      "Requirement already satisfied: termcolor>=1.1.0 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from tensorflow==1.12.0->tensorflowjs) (1.1.0)\n",
      "INFO: pip is looking at multiple versions of six to determine which version is compatible with other requirements. This could take a while.\n",
      "INFO: pip is looking at multiple versions of <Python from Requires-Python> to determine which version is compatible with other requirements. This could take a while.\n",
      "INFO: pip is looking at multiple versions of numpy to determine which version is compatible with other requirements. This could take a while.\n",
      "INFO: pip is looking at multiple versions of keras to determine which version is compatible with other requirements. This could take a while.\n",
      "INFO: pip is looking at multiple versions of h5py to determine which version is compatible with other requirements. This could take a while.\n",
      "INFO: pip is looking at multiple versions of tensorflowjs to determine which version is compatible with other requirements. This could take a while.\n",
      "Collecting tensorflowjs\n",
      "  Downloading tensorflowjs-0.6.7-py3-none-any.whl (34 kB)\n",
      "  Downloading tensorflowjs-0.6.5-py3-none-any.whl (33 kB)\n",
      "  Downloading tensorflowjs-0.6.4-py3-none-any.whl (33 kB)\n",
      "  Downloading tensorflowjs-0.6.2-py3-none-any.whl (33 kB)\n",
      "  Downloading tensorflowjs-0.6.1-py3-none-any.whl (33 kB)\n",
      "  Downloading tensorflowjs-0.6.0-py3-none-any.whl (33 kB)\n",
      "  Downloading tensorflowjs-0.5.7-py3-none-any.whl (33 kB)\n",
      "Collecting tensorflow-hub==0.1.0\n",
      "  Downloading tensorflow_hub-0.1.0-py2.py3-none-any.whl (53 kB)\n",
      "Collecting keras==2.1.6\n",
      "  Downloading Keras-2.1.6-py2.py3-none-any.whl (339 kB)\n",
      "Collecting numpy==1.14.1\n",
      "  Downloading numpy-1.14.1.zip (4.9 MB)\n",
      "  Preparing metadata (setup.py): started\n",
      "  Preparing metadata (setup.py): finished with status 'done'\n",
      "Collecting tensorflowjs\n",
      "  Downloading tensorflowjs-0.5.6-py3-none-any.whl (33 kB)\n",
      "  Downloading tensorflowjs-0.5.4-py3-none-any.whl (33 kB)\n",
      "  Downloading tensorflowjs-0.5.2-py3-none-any.whl (31 kB)\n",
      "  Downloading tensorflowjs-0.5.0-py3-none-any.whl (31 kB)\n",
      "  Downloading tensorflowjs-0.4.2-py3-none-any.whl (31 kB)\n",
      "  Downloading tensorflowjs-0.4.1-py3-none-any.whl (31 kB)\n",
      "  Downloading tensorflowjs-0.4.0-py3-none-any.whl (31 kB)\n",
      "  Downloading tensorflowjs-0.3.1-py3-none-any.whl (30 kB)\n",
      "  Downloading tensorflowjs-0.3.0-py3-none-any.whl (29 kB)\n",
      "  Downloading tensorflowjs-0.2.1-py3-none-any.whl (28 kB)\n",
      "  Downloading tensorflowjs-0.2.0-py3-none-any.whl (28 kB)\n",
      "  Downloading tensorflowjs-0.1.2-py3-none-any.whl (24 kB)\n",
      "  Downloading tensorflowjs-0.1.1-py3-none-any.whl (20 kB)\n",
      "Requirement already satisfied: keras>=2.1.4 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from tensorflowjs) (2.7.0)\n",
      "Requirement already satisfied: keras-applications>=1.0.6 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from tensorflow==1.12.0->tensorflowjs) (1.0.8)\n",
      "Requirement already satisfied: keras-preprocessing>=1.0.5 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from tensorflow==1.12.0->tensorflowjs) (1.1.2)\n",
      "Requirement already satisfied: absl-py>=0.1.6 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from tensorflow==1.12.0->tensorflowjs) (1.0.0)\n",
      "Requirement already satisfied: werkzeug>=0.11.10 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from tensorboard<1.13.0,>=1.12.0->tensorflow==1.12.0->tensorflowjs) (0.16.0)\n",
      "Requirement already satisfied: markdown>=2.6.8 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from tensorboard<1.13.0,>=1.12.0->tensorflow==1.12.0->tensorflowjs) (3.3.6)\n",
      "Requirement already satisfied: importlib-metadata>=4.4 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from markdown>=2.6.8->tensorboard<1.13.0,>=1.12.0->tensorflow==1.12.0->tensorflowjs) (4.8.2)\n",
      "Requirement already satisfied: typing-extensions>=3.6.4 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from importlib-metadata>=4.4->markdown>=2.6.8->tensorboard<1.13.0,>=1.12.0->tensorflow==1.12.0->tensorflowjs) (4.0.0)\n",
      "Requirement already satisfied: zipp>=0.5 in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from importlib-metadata>=4.4->markdown>=2.6.8->tensorboard<1.13.0,>=1.12.0->tensorflow==1.12.0->tensorflowjs) (0.6.0)\n",
      "Requirement already satisfied: more-itertools in c:\\users\\tanu\\anaconda3\\lib\\site-packages (from zipp>=0.5->importlib-metadata>=4.4->markdown>=2.6.8->tensorboard<1.13.0,>=1.12.0->tensorflow==1.12.0->tensorflowjs) (7.2.0)\n",
      "Installing collected packages: tensorflowjs\n",
      "Successfully installed tensorflowjs-0.1.1\n"
     ]
    }
   ],
   "source": [
    "!pip install tensorflowjs\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "ename": "ImportError",
     "evalue": "Traceback (most recent call last):\n  File \"C:\\Users\\TANU\\Anaconda3\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow_internal.py\", line 18, in swig_import_helper\n    fp, pathname, description = imp.find_module('_pywrap_tensorflow_internal', [dirname(__file__)])\n  File \"C:\\Users\\TANU\\Anaconda3\\lib\\imp.py\", line 296, in find_module\n    raise ImportError(_ERR_MSG.format(name), name=name)\nImportError: No module named '_pywrap_tensorflow_internal'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"C:\\Users\\TANU\\Anaconda3\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow.py\", line 58, in <module>\n    from tensorflow.python.pywrap_tensorflow_internal import *\n  File \"C:\\Users\\TANU\\Anaconda3\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow_internal.py\", line 28, in <module>\n    _pywrap_tensorflow_internal = swig_import_helper()\n  File \"C:\\Users\\TANU\\Anaconda3\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow_internal.py\", line 20, in swig_import_helper\n    import _pywrap_tensorflow_internal\nModuleNotFoundError: No module named '_pywrap_tensorflow_internal'\n\n\nFailed to load the native TensorFlow runtime.\n\nSee https://www.tensorflow.org/install/errors\n\nfor some common reasons and solutions.  Include the entire stack trace\nabove this error message when asking for help.",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mImportError\u001b[0m                               Traceback (most recent call last)",
      "\u001b[1;32m~\\Anaconda3\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow_internal.py\u001b[0m in \u001b[0;36mswig_import_helper\u001b[1;34m()\u001b[0m\n\u001b[0;32m     17\u001b[0m         \u001b[1;32mtry\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m---> 18\u001b[1;33m             \u001b[0mfp\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mpathname\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mdescription\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mimp\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mfind_module\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m'_pywrap_tensorflow_internal'\u001b[0m\u001b[1;33m,\u001b[0m \u001b[1;33m[\u001b[0m\u001b[0mdirname\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0m__file__\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m     19\u001b[0m         \u001b[1;32mexcept\u001b[0m \u001b[0mImportError\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\Anaconda3\\lib\\imp.py\u001b[0m in \u001b[0;36mfind_module\u001b[1;34m(name, path)\u001b[0m\n\u001b[0;32m    295\u001b[0m     \u001b[1;32melse\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 296\u001b[1;33m         \u001b[1;32mraise\u001b[0m \u001b[0mImportError\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0m_ERR_MSG\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mformat\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mname\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mname\u001b[0m\u001b[1;33m=\u001b[0m\u001b[0mname\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    297\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;31mImportError\u001b[0m: No module named '_pywrap_tensorflow_internal'",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[1;31mModuleNotFoundError\u001b[0m                       Traceback (most recent call last)",
      "\u001b[1;32m~\\Anaconda3\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow.py\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m     57\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m---> 58\u001b[1;33m   \u001b[1;32mfrom\u001b[0m \u001b[0mtensorflow\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mpython\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mpywrap_tensorflow_internal\u001b[0m \u001b[1;32mimport\u001b[0m \u001b[1;33m*\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m     59\u001b[0m   \u001b[1;32mfrom\u001b[0m \u001b[0mtensorflow\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mpython\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mpywrap_tensorflow_internal\u001b[0m \u001b[1;32mimport\u001b[0m \u001b[0m__version__\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\Anaconda3\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow_internal.py\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m     27\u001b[0m             \u001b[1;32mreturn\u001b[0m \u001b[0m_mod\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m---> 28\u001b[1;33m     \u001b[0m_pywrap_tensorflow_internal\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mswig_import_helper\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m     29\u001b[0m     \u001b[1;32mdel\u001b[0m \u001b[0mswig_import_helper\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\Anaconda3\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow_internal.py\u001b[0m in \u001b[0;36mswig_import_helper\u001b[1;34m()\u001b[0m\n\u001b[0;32m     19\u001b[0m         \u001b[1;32mexcept\u001b[0m \u001b[0mImportError\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m---> 20\u001b[1;33m             \u001b[1;32mimport\u001b[0m \u001b[0m_pywrap_tensorflow_internal\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m     21\u001b[0m             \u001b[1;32mreturn\u001b[0m \u001b[0m_pywrap_tensorflow_internal\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;31mModuleNotFoundError\u001b[0m: No module named '_pywrap_tensorflow_internal'",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[1;31mImportError\u001b[0m                               Traceback (most recent call last)",
      "\u001b[1;32m<ipython-input-2-9ccd7b008696>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[1;32m----> 1\u001b[1;33m \u001b[1;32mfrom\u001b[0m \u001b[0mkeras\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mmodels\u001b[0m \u001b[1;32mimport\u001b[0m \u001b[0mSequential\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m      2\u001b[0m \u001b[1;32mfrom\u001b[0m \u001b[0mkeras\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mlayers\u001b[0m \u001b[1;32mimport\u001b[0m \u001b[0mDense\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      3\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      4\u001b[0m \u001b[1;32mfrom\u001b[0m \u001b[0mtensorflow\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mkeras\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mwrapper\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mscikit_learn\u001b[0m \u001b[1;32mimport\u001b[0m \u001b[0mKerasClassifier\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      5\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\Anaconda3\\lib\\site-packages\\keras\\__init__.py\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m     19\u001b[0m \"\"\"\n\u001b[0;32m     20\u001b[0m \u001b[1;31m# pylint: disable=unused-import\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m---> 21\u001b[1;33m \u001b[1;32mfrom\u001b[0m \u001b[0mtensorflow\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mpython\u001b[0m \u001b[1;32mimport\u001b[0m \u001b[0mtf2\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m     22\u001b[0m \u001b[1;32mfrom\u001b[0m \u001b[0mkeras\u001b[0m \u001b[1;32mimport\u001b[0m \u001b[0mdistribute\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     23\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\Anaconda3\\lib\\site-packages\\tensorflow\\__init__.py\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m     22\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     23\u001b[0m \u001b[1;31m# pylint: disable=g-bad-import-order\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m---> 24\u001b[1;33m \u001b[1;32mfrom\u001b[0m \u001b[0mtensorflow\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mpython\u001b[0m \u001b[1;32mimport\u001b[0m \u001b[0mpywrap_tensorflow\u001b[0m  \u001b[1;31m# pylint: disable=unused-import\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m     25\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     26\u001b[0m \u001b[1;32mtry\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\Anaconda3\\lib\\site-packages\\tensorflow\\python\\__init__.py\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m     47\u001b[0m \u001b[1;32mimport\u001b[0m \u001b[0mnumpy\u001b[0m \u001b[1;32mas\u001b[0m \u001b[0mnp\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     48\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m---> 49\u001b[1;33m \u001b[1;32mfrom\u001b[0m \u001b[0mtensorflow\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mpython\u001b[0m \u001b[1;32mimport\u001b[0m \u001b[0mpywrap_tensorflow\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m     50\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     51\u001b[0m \u001b[1;32mfrom\u001b[0m \u001b[0mtensorflow\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mpython\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mtools\u001b[0m \u001b[1;32mimport\u001b[0m \u001b[0mcomponent_api_helper\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\Anaconda3\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow.py\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m     72\u001b[0m \u001b[1;32mfor\u001b[0m \u001b[0msome\u001b[0m \u001b[0mcommon\u001b[0m \u001b[0mreasons\u001b[0m \u001b[1;32mand\u001b[0m \u001b[0msolutions\u001b[0m\u001b[1;33m.\u001b[0m  \u001b[0mInclude\u001b[0m \u001b[0mthe\u001b[0m \u001b[0mentire\u001b[0m \u001b[0mstack\u001b[0m \u001b[0mtrace\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     73\u001b[0m above this error message when asking for help.\"\"\" % traceback.format_exc()\n\u001b[1;32m---> 74\u001b[1;33m   \u001b[1;32mraise\u001b[0m \u001b[0mImportError\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mmsg\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m     75\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     76\u001b[0m \u001b[1;31m# pylint: enable=wildcard-import,g-import-not-at-top,unused-import,line-too-long\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;31mImportError\u001b[0m: Traceback (most recent call last):\n  File \"C:\\Users\\TANU\\Anaconda3\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow_internal.py\", line 18, in swig_import_helper\n    fp, pathname, description = imp.find_module('_pywrap_tensorflow_internal', [dirname(__file__)])\n  File \"C:\\Users\\TANU\\Anaconda3\\lib\\imp.py\", line 296, in find_module\n    raise ImportError(_ERR_MSG.format(name), name=name)\nImportError: No module named '_pywrap_tensorflow_internal'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"C:\\Users\\TANU\\Anaconda3\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow.py\", line 58, in <module>\n    from tensorflow.python.pywrap_tensorflow_internal import *\n  File \"C:\\Users\\TANU\\Anaconda3\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow_internal.py\", line 28, in <module>\n    _pywrap_tensorflow_internal = swig_import_helper()\n  File \"C:\\Users\\TANU\\Anaconda3\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow_internal.py\", line 20, in swig_import_helper\n    import _pywrap_tensorflow_internal\nModuleNotFoundError: No module named '_pywrap_tensorflow_internal'\n\n\nFailed to load the native TensorFlow runtime.\n\nSee https://www.tensorflow.org/install/errors\n\nfor some common reasons and solutions.  Include the entire stack trace\nabove this error message when asking for help."
     ]
    }
   ],
   "source": [
    "from keras.models import Sequential\n",
    "from keras.layers import Dense\n",
    "\n",
    "from tensorflow.keras.wrapper.scikit_learn import KerasClassifier\n",
    "\n",
    "def createModel():\n",
    "    model = Sequential()\n",
    "    model.add(Dense(6,input_dim=4,kernel_initializer='normal',activation='relu'))\n",
    "    model.add(Dense(4,kernel_initializer='normal',activation='relu'))\n",
    "    model.add(Dense(loss='binary_crossentropy',optimizer='adam',metrics=['accuracy']))\n",
    "    \n",
    "    return model\n",
    "\n",
    "\n",
    "estimator = KerasClassifier(build_fn=createModel,epochs=50,batch_size=10,verbose=0)\n",
    "\n",
    "cv_score = cross_val_score(estimator,features,target,cv=10)\n",
    "\n",
    "print(cv_score.mean())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
